pkg/cloud/fakes.go

```diff
@@ -114,26 +114,13 @@ func (c *FakeCloudProvider) WaitForFileSystemResize(ctx context.Context, fileSys
 func (c *FakeCloudProvider) CreateVolume(ctx context.Context, volumeName string, volumeOptions VolumeOptions) (*Volume, error) {
 	v, exists := c.volumes[volumeName]
 	if exists {
-		if v.StorageCapacityReservationGiB == *volumeOptions.StorageCapacityReservationGiB {
-			return v, nil
-		} else {
-			return nil, ErrAlreadyExists
-		}
-	}
-
-	var storageCapacity int64
-	if volumeOptions.StorageCapacityQuotaGiB == nil {
-		storageCapacity = 10
-	} else {
-		storageCapacity = *volumeOptions.StorageCapacityQuotaGiB
+		return v, nil
```

Comment:
I'm not sure if this is best practice, but since we no longer return an accurate capacity to the RPC there's nothing to cross reference the retrieved volume against. Let me know if you think this should be implemented differently.

Reply:
Side note: You might get a false negative in the sanity tests since we are disabling volume expansion only for volumes, and our controller says we support the capability. If so, let me know the test name and I can ignore it in my PR.

Reply:
Another side node: Fixing up this class may be out of scope for you. I can do it in my e2e PR. For the time being you can just `return v, nil` and I'll take a look in fixing up this class and any failed sanity tests.